src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs

```diff
@@ -511,6 +538,16 @@ internal static class ResourceFilter
                 "source",
             };
 
+            public static readonly List<string> _noPatientTemplate = new List<string>
+            {
+                "ADT_A40", "ADT_A41", "ADT_A45", "ADT_A47",
+            };
+
+            public static readonly List<string> _multiplePatientTemplate = new List<string>
```

Comment:
Same for this one.

Reply:
Sure , I will make it private and push the changes